src/main/java/org/jabref/gui/maintable/MainTable.java

```diff
@@ -193,6 +203,31 @@ public MainTable(MainTableDataModel model,
 
         this.setItems(model.getEntriesFilteredAndSorted());
 
+        Button addExampleButton = new Button(Localization.lang("Add example entry"));
+        addExampleButton.getStyleClass().add("text-button-blue");
+        addExampleButton.setOnAction(event -> {
+            BibEntry entry = addExampleEntry();
+            libraryTab.showAndEdit(entry);
+        });
+
+        Button importPdfsButton = new Button(Localization.lang("Import existing PDFs"));
+        importPdfsButton.getStyleClass().add("text-button-blue");
+        importPdfsButton.setOnAction(event -> importPdfs());
+
+        Label noContentLabel = new Label(Localization.lang("No content in table"));
+
+        HBox buttonBox = new HBox(20, addExampleButton, importPdfsButton);
+        buttonBox.setAlignment(Pos.CENTER);
+
+        VBox placeholderBox = new VBox(15, noContentLabel, buttonBox);
+        placeholderBox.setAlignment(Pos.CENTER);
+
+        updatePlaceholder(placeholderBox);
+
+        database.getDatabase().getEntries().addListener((ListChangeListener<BibEntry>) change -> updatePlaceholder(placeholderBox));
```

Comment:
Does following work, too? - I don't see any usage of the variable.

```suggestion
        database.getDatabase().getEntries().addListener(_ -> updatePlaceholder(placeholderBox));
```

Reply:
I'm sticking with the original code since the lambda with an underscore parameter creates ambiguity error.